src/beamlime/config/raw_detectors/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+This config is used during Live Data Reduction detector view.
+Currently the instrument specific config is stored in python files, but
+they can be moved to a separate file format in the future.
+"""
+
+from .dream import _dream
+from .loki import _loki
+from .nmx import _nmx
+
+__all__ = ["_dream", "_loki", "_nmx"]
```

Comment:
Should we consider it public now and remove the leading `_`?

Reply:
I would then argue to make it something like `dream_detectors_config`, so we don't litter around a python object `dream` in the python namespace.